ComfySigns/Patches/ZoneSystemPatch.cs

```diff
@@ -0,0 +1,20 @@
+﻿using HarmonyLib;
+
+using ComfyLib;
+
+using static ComfySigns.ComfySigns;
+using static ComfySigns.PluginConfig;
+
+namespace ComfySigns.Patches {
+  [HarmonyPatch(typeof(ZoneSystem))]
+  public class ZoneSystemPatch {
```

Comment:
`static class ZoneSystemPatch {`

Reasons for this:

  - Ensures no instance fields in patch classes.
  - Ensures no access to static fields from outside the patch class.

Second point I might revisit in the future.